### PR TITLE
docs(api.md): Comply with our coding style.

### PR DESCRIPTION
**Pull request by hristoterezov**
Based on https://github.com/jitsi/jitsi-meet/pull/4002 in order to update the devices handling examples as well.

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -34,49 +34,54 @@ Its constructor gets a number of options:
...
 ```javascript
-var domain = "meet.jit.si";
-var options = {
+const domain = 'meet.jit.si';
+const options = {
     ...
     devices: {
         'audioInput': '<deviceLabel>',
         'audioOutput': '<deviceLabel>',
         'videoInput': '<deviceLabel>'
```
Comment (virtuacoplenny) [inline doc/api.md]: If we're gonna take shots at styling, the quotes around the keys are unnecessary and, more importantly, inconsistently used within this file.

File: doc/api.md [hunk 2]
```diff
@@ -327,60 +335,60 @@ NOTE: The devices object has the same format as the getAvailableDevices result f
...
-"email": email // the new email
+    'id': id, // the id of the participant that changed his email
+    'email': email // the new email
 }
 ```
 * **filmstripDisplayChanged** - event notifications about the visibility of the filmstrip being updated.
 ```javascript
 {
-"visible": visible, // Whether or not the filmstrip is displayed or hidden.
+    'visible': visible // Whether or not the filmstrip is displayed or hidden.
```
Comment (virtuacoplenny) [inline doc/api.md]: You could be really strict about following coding styling and use property shorthands.

File: doc/api.md (hunk 1 above)
Comment (saghul) [reply doc/api.md]: agreed

File: doc/api.md (hunk 2 above)
Comment (hristoterezov) [reply doc/api.md]: Here we're providing example of the structure. Isn't the shorthand gonna be a little confusing?

How about maybe renaming the `visible` value variable to something else? 

Or maybe to change it to:
```
{
    visible: boolean
}
```

WDYT?

Comment (virtuacoplenny) [reply doc/api.md]: Better

File: doc/api.md (hunk 1 above)
Comment (hristoterezov) [reply doc/api.md]: Fixed!

File: doc/api.md (hunk 2 above)
Comment (hristoterezov) [reply doc/api.md]: Fixed!

**Pull request merged**